lib/database/adapters/TagAdapter.js

```diff
@@ -29,13 +29,15 @@ class TagAdapter {
      * @param {SequelizeTag} databaseObject Object to convert.
      * @returns {Tag} Converted entity object.
      */
-    toEntity({ id, text, email, mattermost, last_edited_name, updatedAt }) {
+    toEntity({ id, text, email, mattermost, last_edited_name, archived, archivedAt, updatedAt }) {
         return {
             id,
             text,
             lastEditedName: last_edited_name,
             email,
             mattermost,
+            archived,
+            archivedAt: archivedAt ? new Date(archivedAt).getTime() : null,
```

Comment:
In the interest of using the sane standard, should this line be written with `||`?

Reply:
We do not use archivedAt "as is" so we cannot do that